build.rs

```diff
@@ -1,40 +1,42 @@
 fn main() {
-    println!("cargo:rerun-if-env-changed=CUSTOM_LIBFUZZER_PATH");
-    if let Ok(custom) = ::std::env::var("CUSTOM_LIBFUZZER_PATH") {
-        println!("cargo:rerun-if-changed={custom}");
+    if cfg!(feature = "link") {
```

Comment:
Can we move all the linking code out to a new function that is also `cfg(feature = "link")` so that it is easier to read this giant `if` with no else?